src/PrestaShopBundle/Resources/config/services/adapter/state.yml

```diff
@@ -0,0 +1,36 @@
+services:
+  _defaults:
+    public: true
+
+  prestashop.adapter.state.command_handler.abstract_state_handler:
+    abstract:  true
+    class: 'PrestaShop\PrestaShop\Adapter\State\AbstractStateHandler'
+
+  prestashop.adapter.state.command_handler.create_state_handler:
+    public: true
+    class: 'PrestaShop\PrestaShop\Adapter\State\CommandHandler\AddStateHandler'
+    parent: 'prestashop.adapter.state.command_handler.abstract_state_handler'
```

Comment:
I don't think you need to configure parent. It's useful when abstract class has dependencies, in your case it doesn't.